package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "drafter",
+  "version": "1.0.0",
+  "description": "Node API Blueprint Parser",
+  "main": "lib/drafter.js",
+  "engines": {
+    "node": ">= 0.12"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/apiaryio/drafter-npm"
+  },
+  "author": "Apiary",
```

Comment:
I believe this should be one of these

- `Apiary Czech Republic, s.r.o. <support@apiary.io>`
- `Apiary Inc. <support@apiary.io>`

according to our internal guidelines, depending for which company the specific package founder (in this case @kylef) works